docs/modules/plugin/nav.adoc

```diff
@@ -0,0 +1,11 @@
+* Maven Plugin
+** xref:introduction.adoc[Introduction]
```

Comment:
You don't have to specify text in the xref module. It will use the page title, which you can override using the `navtitle` attribute defined in the document header of that page.